cli/Makefile

```diff
@@ -18,8 +18,8 @@ build:
 
 .PHONY: publish
 publish:
-	gsutil cp "${BUILD_DIR}/*" "${BUCKET}/${VERSION}"
-	gsutil cp "${BUILD_DIR}/*" "${BUCKET}/latest"
+	gcloud alpha storage cp "${BUILD_DIR}/*" "${BUCKET}/${VERSION}"
```

Comment:
This is a known limitation
https://github.com/google-github-actions/auth#configuring-gcloud

Reply:
That's fine, I prefer gcloud anyways.